common/djangoapps/student/tests/test_admin_views.py

```diff
@@ -165,7 +165,14 @@ def setUp(self):
         super(AdminUserPageTest, self).setUp()
         self.admin = UserAdmin(User, AdminSite())
 
-    def test_username_is_readonly(self):
+    def test_username_is_writable_for_user_creation(self):
+        """
+        Ensures that the username is not readonly, when admin creates new user.
+        """
+        request = Mock()
+        self.assertNotIn('username', self.admin.get_readonly_fields(request))
+
+    def test_username_is_readonly_for_user(self):
         """
         Ensures that the username is readonly to skip Django validation in the `auth_user_change` view.
```

Comment:
Please add description for this test case to make it clear what is the difference between test_username_is_writable_for_user_creation and test_username_is_readonly_for_user tests.